omod/src/main/webapp/resources/jsonforms/HMIS-TB-004-REQUEST-FORM-FOR-TB-SPECIMEN-EXAMINATION.json

```diff
@@ -0,0 +1,192 @@
+{
+  "encounterType": "08478ad9-ccc1-4cbe-9e55-473447984158",
+  "name": "HMIS TB 002: REQUEST FORM FOR TB SPECIMEN EXAMINATION",
+  "processor": "EncounterFormProcessor",
+  "referencedForms": [],
```

Comment:
@jabahum i see missing attributes like encounter, published and retire

Reply:
Resolved